README.md

```diff
@@ -53,23 +53,28 @@ Otherwise you will get an error like this
 * If you make changes in code and nothing changes you can try to uninstall the plugin:
 ```
 pip3 uninstall actinia-example-plugin.wsgi -y
+rm -rf /usr/lib/python3.8/site-packages/actinia_example_plugin.wsgi-*.egg
```

Comment:
Did you try
```suggestion
pip3 uninstall actinia_example_plugin.wsgi -y
```
?

Reply:
Yes for the actinia-tiling-plugin I also tested this and only removing the folder was helping.

Reply:
-.- ok then pls leave as is.